File: C_Programming/Assignment/Assignment3/palindrome.c
#include<stdio.h>
void main()
{
    int num,rev=0,rem,temp;
    printf("Enter an integer:");
    scanf("%d",&num);
    temp=num;
    while(num!=0)
    {
    rem=num%10;
    rev=rev*10+rem;
    num=num/10;
}
if(temp==rev)
{
    printf("%d is a palindrome",temp);

}
else
{
     printf(" %d is not a palindrome",temp);
}

}

//output:Enter an integer:121
//121 is a palindrome
//Enter an integer:2345
//2345 is not a palindrome

File: Java_Programs/Assignment3/Palindrome.java
package Assignment3;

public class Palindrome {

	public static void main(String[] args) {
		int num=121,rev=0,rem,temp;
//	    printf("Enter an integer:");
//	    scanf("%d",&num);
	    temp=num;
	    while(num!=0)
	    {
	    rem=num%10;
	    rev=rev*10+rem;
	    num=num/10;
	}
	if(temp==rev)
	{
	    System.out.printf("%d is a palindrome",temp);

	}
	else
	{
	     System.out.printf(" %d is not a palindrome",temp);
	}
	}

}
